### PR TITLE
Added release notes for release 1.23.1

### PR DESCRIPTION
**Pull request by ifbyol**
Release notes for `1.23.1`

### CODE REVIEW

File: src/content/release-notes.mdx [hunk 1]
```diff
@@ -5,6 +5,16 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.23.1
+
+14 August 2024
+
+This versions is compatible with Kubernetes versions 1.26 to 1.30
```
Comment (codyjlandstrom) [inline src/content/release-notes.mdx:12]:
```suggestion
This version is compatible with Kubernetes versions 1.26 to 1.30
```

File: src/content/release-notes.mdx [hunk 2]
```diff
@@ -5,6 +5,16 @@ sidebar_label: Release notes
...
+
+### Bug Fixes
+
+- Updated the backend CLI to 2.30.1 to fix an error when deploying remotely an Okteto Manifest storing a variable with the symbol `=` on [$OKTETO_ENV](core/okteto-variables.mdx##okteto_env)
```
Comment (codyjlandstrom) [inline src/content/release-notes.mdx:16]:
```suggestion
- Updated the backend CLI to 2.30.1 to fix an error when deploying remotely an Okteto Manifest storing a variable with the symbol `=` on [$OKTETO_ENV](core/okteto-variables.mdx#okteto_env)
```

File: versioned_docs/version-1.23/release-notes.mdx [hunk 3]
```diff
@@ -5,6 +5,16 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.23.1
+
+14 August 2024
+
+This versions is compatible with Kubernetes versions 1.26 to 1.30
```
Comment (codyjlandstrom) [inline versioned_docs/version-1.23/release-notes.mdx:12]:
```suggestion
This version is compatible with Kubernetes versions 1.26 to 1.30
```

File: versioned_docs/version-1.23/release-notes.mdx [hunk 4]
```diff
@@ -5,6 +5,16 @@ sidebar_label: Release notes
...
+
+### Bug Fixes
+
+- Updated the backend CLI to 2.30.1 to fix an error when deploying remotely an Okteto Manifest storing a variable with the symbol `=` on [$OKTETO_ENV](core/okteto-variables.mdx##okteto_env)
```
Comment (codyjlandstrom) [inline versioned_docs/version-1.23/release-notes.mdx:16]:
```suggestion
- Updated the backend CLI to 2.30.1 to fix an error when deploying remotely an Okteto Manifest storing a variable with the symbol `=` on [$OKTETO_ENV](core/okteto-variables.mdx#okteto_env)
```

**Comment by ifbyol** [comment]
Release was not done yet, but leaving the PR ready to be merged as soon as we do the release

**Review by codyjlandstrom — APPROVED** [review]

**Pull request merged**